### DISCUSSION TITLE
Following up on new releases

### DISCUSSION THREAD

**Discussion by o-alquimista** [brgl/libgpiod#82] [General]
I'm building from source, since my distribution ships a really old version of this (pre-v2), and it would be good to be notified of new releases (tags).

The mailing list is way too busy, so I had to unsubscribe. This github mirror sadly doesn't sync tags from git.kernel.org, or I could use the "watch" feature. Any other way? RSS would be great too (gitlab has that).

**Comment by brgl** [comment]
I don't want to use github to distribute tarballs. I prefer to stay with kernel.org. Is there any way of notifying users about new releases? If I push tags to github, does it send any notifications?

**Comment by Ablu** [comment]
> RSS would be great too (gitlab has that).

If you have a feed reader you can create a filter on top public-inbox and get the (atom) feed.

Example: https://lore.kernel.org/linux-gpio/?q=%22%5BANNOUNCE%5D%22+libgpiod+-%22Re%3A%22
Feed: https://lore.kernel.org/linux-gpio/?q=%22%5BANNOUNCE%5D%22+libgpiod+-%22Re:%22&x=A

(assuming @brgl keeps his announcement mail pattern)

**Reply by o-alquimista** [reply]
Brilliant. Thanks!

![Screenshot_2024-06-14_18-14-45](https://github.com/brgl/libgpiod/assets/25424263/0e7bff2c-a04a-4d60-bbb2-f48ff802c528)
